cfgov/regulations3k/wagtail_hooks.py

```diff
@@ -45,6 +48,34 @@ class EffectiveVersionModelAdmin(TreeModelAdmin):
     child_model_admin = SubpartModelAdmin
     parent_field = 'part'
 
+    def copy(self, instance):
+        # Create a new instance
+        new_version = EffectiveVersion(
+            authority=instance.authority,
+            source=instance.source,
+            part=instance.part,
+            effective_date=date.today(),
```

Comment:
I would vote for `instance.effective_date` as a starting point. 
'today()' will almost never be right, although it might be closest in time to a prospective new effective_date.

I worry that it might be confusing to editors if `acquired_date` and `effective_date` pop up as the same.

Reply:
Avoiding confusion was my motivation not to use the same `effective_date`, because that's currently the unique identifier for a human looking at the list of versions.

Created might be better, because then that becomes the distinguishing field for telling the difference between the version you copied and the new copy.

Reply:
It is hard to imagine a common use case for creating a new tree with the same effective date, so copying the old date would almost always be wrong, too. I don't object to today() as a starting point.

Reply:
Your changes in #4387 pretty much resolve all my concerns about clarity of what you've just done by clicking "Copy". We'll keep the effective date the same and update the created date.